cmd/exporter/main.go

```diff
@@ -49,14 +49,23 @@ func main() {
 	ctx, cancel := context.WithCancel(context.Background())
 	go func() {
 		defer cancel()
-		scrape, err := gocdexporter.NewScraper(&gocdexporter.Config{
+		conf := &gocdexporter.Config{
 			*ns, prometheus.DefaultRegisterer, *url, *user, *passwd, *agentMaxPages,
-		})
+		}
+		scrape, err := gocdexporter.NewScraper(conf)
 		if err != nil {
 			log.Fatal(err)
 		}
 		ticker := time.NewTicker(dur)
 		defer ticker.Stop()
+		errorGauge := prometheus.NewGaugeVec(
+			prometheus.GaugeOpts{
+				Namespace: conf.Namespace,
+				Name:      "scrape_error",
+				Help:      "prometheus-gocd-exporter scrape error",
+			},
+			[]string{})
+		conf.Registerer.Register(errorGauge)
```

Comment:
could use `prometheus.DefaultRegisterer` directly instead of referencing from conf to make it clear it's not really used inside Scrape. OR move the gauge to inside the scraper and count [here](https://github.com/pagero/prometheus-gocd-exporter/blob/master/scraper.go#L67)